resources/js/components/assets/Browser/Browser.vue

```diff
@@ -479,14 +480,6 @@ export default {
             this.$progress.loading('asset-browser', loading);
         },
 
-        editedAssetId(editedAssetId) {
-            let path = editedAssetId
-                ? [this.path, this.editedAssetBasename].filter(value => value != '/').join('/') + '/edit'
-                : this.path;
-
-            this.$emit('navigated', this.container, path);
-        },
-
```

Comment:
`editedAssetId` was being used, now it's deleted? 🤔

Reply:
This was just the watcher. It's no longer needed since we're emitting the `navigated` event from the `AssetEditor` component which does the AJAX request to get the `edit_url`.

Reply:
🤦‍♂️ Yeah. 😅